integration/shared/scenarios/export_service.go

```diff
@@ -82,7 +82,7 @@ func (e *exportServiceScenario) Run() error {
 
 	return wait.Poll(defaultScenarioPollInterval, defaultScenarioPollTimeout, func() (done bool, err error) {
 		fmt.Println("Polling service...")
-		cmSvc, err := e.sdClient.GetService(context.TODO(), e.expectedSvc.Namespace, e.expectedSvc.Name)
+		cmSvc, err := e.sdClient.GetService(context.TODO(), e.expectedSvc.Namespace, e.expectedSvc.Name, "")
```

Comment:
Unsure what to do here.

Reply:
Tests will currently succeed since clusterset id is presumably being exported as blank during test. Once cluster id and cluster set id is required kind/eks test setups will need to set this in the test clusters. The `exportServiceScenario` object should get a clusterset id field and pass that value to `GetService`.